docs/c-ffi/calling-c.md

```diff
@@ -80,7 +80,8 @@ use @eglGetDisplay[Pointer[_EGLDisplayHandle]](disp: Pointer[_XDisplayHandle])
 primitive _XDisplayHandle
 primitive _EGLDisplayHandle
 
-let x_dpy = @XOpenDisplay(Pointer[U8])
+let x_dpy_name: String = // ...
```

Comment:
So the code here already "doesn't compile" as it isn't valid. I don't think this change makes things better.

I think that this could be slightly better if the string is given a realistic display name.
This could be made "much better" by taking all the C calling code and make it "real" by putting the "FFI code examples" in code that would at least compile.